tests/src/test/scala/common/TestUtils.java

```diff
@@ -69,7 +69,7 @@
     public static final int CONFLICT        = 153;      // 409 - 256 = 153
     public static final int TOO_LARGE       = 157;      // 413 - 256 = 157
     public static final int THROTTLED       = 173;      // 429 (TOO_MANY_REQUESTS) - 256 = 173
-    public static final int APP_ERROR       = 246;      // 502 - 256 = 246
+    public static final int APP_ERROR       = 244;      // 500 - 256 = 244
```

Comment:
Why does this need to be changed?

Reply:
APP_ERROR and TIMEOUT have same error code. So I couldn't figure out exactly what kind of error happened to controller after kafka shooting test done.
Anyway it is unrelated with this PR, I'm gonna rollback it.